interfaces/default/js/default.js

```diff
@@ -267,3 +267,9 @@ $('.dropdown-toggle').click(function(e) {
     }
   }, this), 0);
 });
+// Pad numbers with leading zero, e.g. S1E1 becomes S01E01, useful for episode numbering
+Number.prototype.pad = function(size) {

```

Comment:
When I tried using the existing pad function the page failed to load.  Should it be used in the same way as the function i added, i.e. by adding `.pad` to the value so `cal.seasonNumber.pad`?  That’s how I tried it.


> On 29 May 2015, at 17:54, Hellowlol <notifications@github.com> wrote:
> 
> In interfaces/default/js/default.js <https://github.com/Hellowlol/HTPC-Manager/pull/195#discussion_r31342613>:
> 
> > @@ -267,3 +267,9 @@ $('.dropdown-toggle').click(function(e) {
> >      }
> >    }, this), 0);
> >  });
> > +// Pad numbers with leading zero, e.g. S1E1 becomes S01E01, useful for episode numbering
> > +Number.prototype.pad = function(size) {
> Can't you use the pad function that already exist?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Hellowlol/HTPC-Manager/pull/195/files#r31342613>.
>